testapp/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,7 @@
         android:name="com.rakuten.tech.mobile.testapp.SampleApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+        android:label="@string/hostapp_name"
```

Comment:
But isn't this going to break the title on the different build variants? For example, all of them will be named "Mini App Sample" instead of "Mini App Sample DEBUG" and "Mini App Sample STG". See https://github.com/rakutentech/android-miniapp/blob/master/testapp/build.gradle#L59

Actually, couldn't you just completely delete `app_name` (and `hostapp_name`) from `strings.xml` because it's being set in `build.gradle`?

Reply:
Yeah, I agree. I have figured out something from https://developer.android.com/studio/build/gradle-tips that we can utilize `manifestPlaceholders`. I have updated with it.